aries-site/src/examples/templates/forms/RequiredFieldsExample.js

```diff
@@ -0,0 +1,172 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  CheckBox,
+  Form,
+  FormField,
+  Header,
+  Select,
+  Text,
+  TextInput,
+  TextArea,
+} from 'grommet';
+import { CircleAlert } from 'grommet-icons';
+
+const superPower = ['Flying', 'Sky Runner', 'Invisibility'];
+const weakness = ['Fire', 'PB & J', 'Kryptonite'];
+
+const RequiredFormField = props => {
+  const { required, label, ...rest } = props;
+  return (
+    <FormField
+      label={required ? `${label}*` : label}
+      required={required}
+      {...rest}
+    />
+  );
+};
+
+RequiredFormField.propTypes = {
+  required: PropTypes.bool,
+  label: PropTypes.string,
+};
+
+const FormContainer = ({ ...rest }) => {
+  return (
+    <Box background="background-front" border round="small" overflow="hidden">
+      <Box flex pad={{ horizontal: 'medium', vertical: 'medium' }} {...rest} />
+    </Box>
+  );
+};
+
+const Error = ({ children, ...rest }) => {
+  return (
+    <Box direction="row" {...rest}>
+      <Box flex={false} margin={{ top: 'xxsmall', right: 'xxsmall' }}>
+        <CircleAlert size="small" />
+      </Box>
+      <Text size="xsmall">{children}</Text>
+    </Box>
```

Comment:
This seems like the Box around the icon is unneeded. Would the following achieve the same result? 

```suggestion
    <Box direction="row" align="center" {...rest}>
        <CircleAlert size="small" />
      <Text size="xsmall">{children}</Text>
    </Box>
```

Either way, the current icon placement seems out of alignment:
![Screen Shot 2020-08-04 at 5 11 24 PM](https://user-images.githubusercontent.com/1756948/89354305-bb4b1880-d675-11ea-8af9-0240414b0beb.png)

Reply:
will open a new issue for this.. going with work around for now!